lrlex/src/lib/parser.rs

```diff
@@ -297,7 +320,13 @@ mod test {
                 kind: LexErrorKind::DuplicateName,
                 line: 3,
                 col: 7,
-            }) => (),
+                span,
+            }) => {
+                // I would suggest that this should actually span around int
```

Comment:
Is this comment saying "the error message isn't great"? If so, we should file this as an issue and fix it later rather than leave a comment.

Reply:
Yeah, more-or-less, I'd go with "this error message could be improved", but I will pull these comments out, and file an issue.